A3A/addons/core/functions/init/fn_initClient.sqf

```diff
@@ -41,6 +41,8 @@ if !(isServer) then {
     call A3A_fnc_initVarCommon;
 
     [] execVM QPATHTOFOLDER(Scripts\fn_advancedTowingInit.sqf);
+    [] execVM QPATHTOFOLDER(Scripts\SpectumDevice\spectrum_device.sqf);
+    [] execVM QPATHTOFOLDER(Scripts\SpectumDevice\sa_ewar.sqf);
```

Comment:
Add a parameter to disable this and it'll probably be fine
```suggestion
    if (enableSpectrumDevice) then {
        [] execVM QPATHTOFOLDER(Scripts\SpectumDevice\spectrum_device.sqf);
        [] execVM QPATHTOFOLDER(Scripts\SpectumDevice\sa_ewar.sqf);
    };
```

Reply:
If you can, confirm that this check works, if so, I'll just merge it